src/core/SuccessResponse.ts

```diff
@@ -0,0 +1 @@
+export type SuccessResponse<T> = T & { _meta: { request: any; response: any } }
```

Comment:
Have we considered making the types stricter here?

I'd go with

```
request: unknown
response: AxiosResponse<unknown, unknown>
```

to avoid doing wrong things to the data silently

Reply:
Or maybe `AxiosResponse<T, unknown>` possibly, since we have `T` in this context